### PR TITLE
Changes to remove sphinx doc build errors.

### PR DESCRIPTION
**Pull request by vmagelo**
Please see comments in files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ The following are a list of the experimental explainers available in the communi
...
+* Mimic Explainer: Mimic explainer is based on the idea of [global surrogate models](https://christophm.github.io/interpretable-ml-book/global.html)'s. A global surrogate model is an intrinsically interpretable model that is trained to approximate the predictions of a black box model as accurately as possible. Data scientist can interpret the surrogate model to draw conclusions about the black box model. This repository supports the following interpretable models as surrogate model: LightGBM (LGBMExplainableModel), Linear/Logistic Regression (LinearExplainableModel), Stochastic Gradient Descent explainable model (SGDExplainableModel), and Decision Tree (DecisionTreeExplainableModel).
```
Comment (vmagelo) [inline README.md]: Hard to see but changes "supporots" to "supports". That's it.

File: python/interpret_community/common/model_wrapper.py [hunk 2]
```diff
@@ -182,7 +182,7 @@ def wrap_model(model, examples, model_task):
         outputs a 1 dimensional array.
     :type model_task: str
     :return: The wrapper model.
-    :rtype model
+    :rtype: interpret_community.mimic.models.explainable_model.BaseExplainableModel.model
```
Comment (vmagelo) [inline python/interpret_community/common/model_wrapper.py]: This was missing closing ":". Then after adding that, the "model" type was ambiguous with the sphinx error show below. The error went away when I specify the exact model. If we are okay with ambiguous error in sphinx build then we can just leave as "model". Thoughs?


Error::

C:\Users\v-magelo\Documents\GitHub\interpret-community\python\interpret_community\common\model_wrapper.py:docstring of interpret_community.common.model_wrapper.wrap_model:None: WARNING: more than one target found for cross-reference 'model': interpret_community.mimic.models.explainable_model.BaseExplainableModel.model,
interpret_community.mimic.models.lightgbm_model.LGBMExplainableModel.model,
interpret_community.mimic.models.linear_model.LinearExplainableModel.model,
interpret_community.mimic.models.linear_model.SGDExplainableModel.model,
interpret_community.mimic.models.tree_model.DecisionTreeExplainableModel.model,
interpret_community.mimic.models.BaseExplainableModel.model,
interpret_community.mimic.models.LGBMExplainableModel.model,
interpret_community.mimic.models.SGDExplainableModel.model,
interpret_community.mimic.models.LinearExplainableModel.model,
interpret_community.mimic.models.DecisionTreeExplainableModel.model

File: python/interpret_community/community/__init__.py [hunk 3]
```diff
@@ -4,6 +4,6 @@
 
 """Module for model interpretability, including feature and class importance for blackbox and greybox models."""
 
-from .tabular_explainer import TabularExplainer
+from interpret_community import TabularExplainer
```
Comment (vmagelo) [inline python/interpret_community/community/__init__.py]: I was trying to remove the sphinx error (shown below) and making this change removed the error. Is this correct? Or should I revert?

**Sphinx error:**
C:\Users\xxx\Documents\GitHub\interpret-community\python\docbuild\interpret_community.community.rst:7: WARNING: autodoc: failed to import module 'interpret_community.community'; the following exception was raised:
Traceback (most recent call last):
  File "c:\users\xxx\appdata\local\programs\python\python37-32\lib\site-packages\sphinx\ext\autodoc.py", line 551, in import_object
    __import__(self.modname)
  File "C:\Users\xxx\Documents\GitHub\interpret-community\python\interpret_community\community\__init__.py", line 7, in <module>
    from .tabular_explainer import TabularExplainer
ModuleNotFoundError: No module named 'interpret_community.community.tabular_explainer'

Comment (imatiach-msft) [reply python/interpret_community/community/__init__.py]: from interpret_community.tabulat_explainer import TabularExplainer

Comment (vmagelo) [reply python/interpret_community/community/__init__.py]: Thanks. Changed.

File: python/interpret_community/common/model_wrapper.py (hunk 2 above)
Comment (wamartin-aml) [reply python/interpret_community/common/model_wrapper.py]: @imatiach-msft

Comment (imatiach-msft) [reply python/interpret_community/common/model_wrapper.py]: I think it should just be interpret_community.mimic.models.explainable_model.BaseExplainableModel, not interpret_community.mimic.models.explainable_model.BaseExplainableModel.model?

Comment (imatiach-msft) [reply python/interpret_community/common/model_wrapper.py]: oh wait, sorry, this is completely wrong. This is referring to any python model.  Just keep it as model if possible.  It's not any of the surrogate models.

**Comment by imatiach-msft** [comment]
ping @vmagelo any ETA on when this is ready to be merged?

File: python/interpret_community/common/model_wrapper.py (hunk 2 above)
Comment (vmagelo) [reply python/interpret_community/common/model_wrapper.py]: Went back to just model for now.

**Comment by vmagelo** [comment]
@imatiach-msft - checks have passed but merge is still blocked even though you gave approval. Also, I don't think I have write access to merge so can you help here too? Thanks.

**Pull request merged**